templates/form.html

```diff
@@ -52,6 +53,27 @@ <h1 class = "title" >Username Availability Checker</h1>
                 </center>
             </div>
         </div>
+        <div id="usernames">
+            <p class="textualbk"> Need a username? </p>
```

Comment:
Is there a reason that there's whitespace on either side of the text instead of:

```suggestion
            <p class="textualbk">Need a username?</p>
```